Tests/SecItemTests.swift

```diff
@@ -27,7 +27,9 @@ class SecItemTests: XCTestCase {
 
     func test_sharedAccessGroupPrefix_findsPrefix() {
         #if os(iOS)
-            XCTAssertEqual(SecItem.sharedAccessGroupPrefix, "9XUJ7M53NG")
+            #if !COCOAPODS
```

Comment:
@dfed not the best but it will do I think for now. You are already getting good coverage from your xcodeproj. Alternatively I can exclude this file also from the test spec.

Reply:
it seems it would be nice if I expand the `app_host` DSL to allow you to specify certain attributes similar to `pod_target_xcconfig` but it will be a while...

Reply:
This is fine I think. Let's add a good comment as to why we're only testing this in the non-cocoapods case.

Reply:
done